modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * applications/connecting_applications_to_services/odc-connecting-an-application-to-a-service-using-the-developer-perspective.adoc
+:_content-type: PROCEDURE
+[id="odc-creating-a-binding-connection-between-components_{context}"]
+= Creating a binding connection between components
+
+You can establish a binding connection with Operator-backed components.
+
+This procedure walks through an example of creating a binding connection between a PostgreSQL Database service and a Spring PetClinic sample application. To create a binding connection with a service that is backed by the PostgreSQL Database Operator, you must first add the Red Hat-provided PostgreSQL Database Operator to the *OperatorHub*, and then install the Operator.
+The PostreSQL Database Operator then creates and manages the `Database` resource, which exposes the binding information in secrets, config maps, status, and spec attributes.
+
+.Prerequisites
+
+* You have created and deployed a Spring PetClinic sample application by using the *Developer* perspective.
+* You have installed {servicebinding-title} from the OperatorHub.
```

Comment:
the {servicebinding-title}

Reply:
Apologies John, commits from other writers (including this file) got included in my PR. Therefore, I do not include this feedback.